deploy/test-environments/fleet_api/src/configuration_fleet.py

```diff
@@ -18,6 +18,14 @@
 import os
 from munch import Munch
 
+CNVM_TAGS = (
+    "Key=division,Value=engineering "
+    "Key=org,Value=security "
+    "Key=team,Value=cloud-security-posture "
+    "Key=project,Value=test-environments"
```

Comment:
Are the spaces necessary?
```suggestion
    "Key=division,Value=engineering"
    "Key=org,Value=security"
    "Key=team,Value=cloud-security-posture"
    "Key=project,Value=test-environments"
```